lib/logging.go

```diff
@@ -0,0 +1,20 @@
+package lib
+
+import (
+	"github.com/op/go-logging"
+	"os"
+)
+
+// Init logger for the lib package
+var logger = GetLogger()
+
+// GetLogger Will initialize a logger
+func GetLogger() *logging.Logger {
+	logger := logging.MustGetLogger("tfswitch")
+
+	logBackend := logging.NewLogBackend(os.Stdout, "", 0)
+	format := logging.MustStringFormatter("%{time:15:04:05.000} %{shortfunc}: %{level:6.6s} %{message}")
```

Comment:
I'm still concerned about padding the `level` item. It introduces unnecessary spacing IMHO.
![image](https://github.com/warrensbox/terraform-switcher/assets/1274789/d63a7dcb-98a4-4a77-a630-7c296519a25f)

Reply:
Removed it

Reply:
It's not too bad:

```
12:11:28.227 InstallableBinLocation: INFO Installing terraform at /Users/warrenv/bin
12:11:28.279 DownloadFromURL: INFO Downloading to: /Users/warrenv/.terraform.versions
12:11:29.339 DownloadFromURL: INFO 25068075 bytes downloaded
12:11:29.738 Install: INFO Switched terraform to version "1.6.4"
```